docs/4_2_co-simulation_api.adoc

```diff
@@ -86,17 +86,17 @@ include::../headers/fmi3FunctionTypes.h[tags=DoStep]
 Only <<fmi3DoStep>> can change the time of a Co-Simulation FMU from the outside (<<advancing-time,time advances internally during `fmi3DoStep`>>).
 Arguments referring to time inherit the unit of the <<independent>> variable.
 
-* [[currentCommunicationPoint,`currentCommunicationPoint`]] `currentCommunicationPoint` is the current communication point of the importer (latexmath:[\mathbf{t}_\mathit{i}]).
+* [[currentCommunicationPoint,`currentCommunicationPoint`]] `currentCommunicationPoint` is the current communication point of the importer (latexmath:[t_\mathit{i}]).
 At the first call of <<fmi3DoStep>>, <<currentCommunicationPoint>> must be equal to the argument <<startTime>> of <<fmi3EnterInitializationMode>>.
 
 * [[communicationStepSize,`communicationStepSize`]] `communicationStepSize` is the communication step size (latexmath:[\mathbf{h}_\mathit{i}]).
 <<communicationStepSize>> must be latexmath:[> 0.0].
-The FMU is expected to compute until time latexmath:[\mathbf{t}_{i+1} = \mathbf{t}_\mathit{i} + \mathbf{h}_\mathit{i}].
+The FMU is expected to compute until time latexmath:[t_{i+1} = t_\mathit{i} + \mathbf{h}_\mathit{i}].
```

Comment:
\mathbf{h}_\mathit{i} ?
 occurs several times

Reply:
Unboldet h in the entire document.